src/state.js

```diff
@@ -1,12 +1,13 @@
 // @flow
+
+export type Point = { description: string, timestamp: string }
+
 export type State = {|
-  hello: string,
-  resources: { [string]: { id: number } }
+  points?: Point[]
```

Comment:
Toto by sme chceli mat ako 
```suggestion
  points: Point[]
```
Z dovodu, aby sme nemuseli vsade checkovat, ci tam tie `points` su. Bude vsak treba upravit vytahovanie z localstorage-u(https://github.com/vacuumlabs/fine-tracker/blob/master/src/configureStore.js#L22 kniznica `redux-localstorage-simple`) tak, aby ak tam aj predtym points neboli, aby vo vytvorenom stave boli - respektive, aby toto fungovalo univerzalne pre lubovolnynovy field v stave.